core-kotlin-modules/core-kotlin-advanced-3/src/test/kotlin/com/baeldung/optimizeWhenStatement/OptimizeReturnStatmentInWhenExpressionUnitTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.baeldung.optimizeWhenStatement
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class OptimizeReturnStatmentInWhenExpressionUnitTest {
+}
```

Comment:
We probably don't need this empty unit test anymore, and we can probably just move the code out of the test folder to the main folder